api.go

```diff
@@ -57,7 +57,7 @@ type API struct {
 	alerts         provider.Alerts
 	silences       provider.Silences
 	config         string
-	configJSON     config.Config
+	configJSON     *config.Config
```

Comment:
The motivation was consistency with the definitions for marshaling/unmarshaling YAML. Doesn't bother me to change the marshal json to have a value receiver type instead of a pointer, and change this back.